network.koesling.shm-modbus.yml

```diff
@@ -0,0 +1,128 @@
+id: network.koesling.shm-modbus
```

Comment:
is this domain yours and simply misconfigured?

Reply:
yes, i fixed it